docs/en/setup/backend/backend-infrastructure-monitoring.md

```diff
@@ -0,0 +1,119 @@
+# VMs monitoring 
```

Comment:
What's the relationship between this file and `backend-vm-monitoring.md` and `backend-k8s-monitoring.md`, this file seems to be a combination of `backend-k8s-monitoring.md` and `backend-vm-monitoring.md` but the other 2 files are still there, shouldn't them be removed?

Reply:
> What's the relationship between this file and `backend-vm-monitoring.md` and `backend-k8s-monitoring.md`, this file seems to be a combination of `backend-k8s-monitoring.md` and `backend-vm-monitoring.md` but the other 2 files are still there, shouldn't them be removed?

I didn't notice this. Let me remove this. This seems a git auto-merge bug.

Reply:
Done, revmoed.